src/unix/mediactrl_gstplayer.cpp

```diff
@@ -178,6 +178,16 @@ static void realize_callback(GtkWidget* widget, wxGStreamerMediaBackend* be)
     gst_player_video_overlay_video_renderer_set_window_handle(GST_PLAYER_VIDEO_OVERLAY_VIDEO_RENDERER(be->m_video_renderer),
                                 wxGtkGetIdFromWidget(widget)
                                 );
+    GdkWindow* window = gtk_widget_get_window(widget);
+    if (strcmp("GdkWaylandWindow", g_type_name(G_TYPE_FROM_INSTANCE(window))) == 0) {
```

Comment:
Trivial style fix:
```suggestion
    if (strcmp("GdkWaylandWindow", g_type_name(G_TYPE_FROM_INSTANCE(window))) == 0)
    {
```

You may apply it if you prefer doing it yourself or I could do it when merging if you'd rather prefer this.